### PR TITLE
Add new publishing repository; Modify repository search order

### PR DESCRIPTION
**Pull request by shoatman**
- Pointed common gradle root project to new feed
- Added new feed as a repository to publish to

### CODE REVIEW

File: common/build.gradle [hunk 1]
```diff
@@ -274,6 +274,14 @@ afterEvaluate {
...
                 }
             }
+            maven {
+                name "vsts-maven-android"
+                url 'https://identitydivision.pkgs.visualstudio.com/IDDP/_packaging/Android/maven/v1'
+                credentials {
+                    username project.vstsUsername
+                    password project.vstsPassword
+                }
+            }
```
Comment (shahzaibj) [inline common/build.gradle:284]: Do we need to do this? Given that we already did in project level build.gradle file

File: common/versioning/version.properties [hunk 2]
```diff
@@ -1,4 +1,5 @@
-#Mon Nov 16 21:58:07 UTC 2020
+#Mon Nov 16 13:39:37 PST 2020
 versionName=3.0.6
 versionCode=1
-latestPatchVersion=50
+latestPatchVersion=51
+
```
Comment (shahzaibj) [inline common/versioning/version.properties:5]: nit: extra line can be removed

File: common/build.gradle (hunk 1 above)
Comment (shoatman) [reply common/build.gradle:284]: Nope.  I just removed it... as it was breaking the VSTS pipeline.

Comment (shahzaibj) [reply common/build.gradle:284]: Okay that was for a different feed...resolving

**Review by shahzaibj — APPROVED** [review]

**Review by iambmelt — APPROVED** [review]

**Pull request merged**